app/views/devise/registrations/edit.slim

```diff
@@ -0,0 +1,45 @@
+.container-fluid
+  .auth
+    h2 Edit
+
+    = simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
+      = f.error_notification
+
+      .form-inputs
+        = f.input :first_name, required: true, input_html: {autocomplete: "name"}, placeholder: "#{}"
+        = f.input :last_name, required: true, input_html: {autocomplete: "last_name"}
+        = f.input :phone_number, input_html: {autocomplete: "phone_number"}
+        = f.input :email, required: true, autofocus: true
+
+        = if devise_mapping.confirmable? && resource.pending_reconfirmation?
+          p Currently waiting confirmation for:
+          =resource.unconfirmed_email
+
+
+        = f.input :password,
+                    hint: "leave it blank if you don't want to change it",
+                    required: false,
+                    input_html: { autocomplete: "new-password" }
+        = f.input :password_confirmation,
+                    required: false,
+                    input_html: { autocomplete: "new-password" }
+        = f.input :current_password,
+                    hint: "we need your current password to confirm your changes",
+                    required: true,
+                    input_html: { autocomplete: "current-password" }
+
+
+      .form-actions
+        = f.button :submit, "Update"
+
+
+
+    h3 Cancel my account
```

Comment:
please remove link to cancel the account because deleting user from db will cause lots of problems with relations